src/components/Login/index.tsx

```diff
@@ -31,40 +38,44 @@ function LoginModalContent(props: Props) {
       <ModalOverlay />
       <ModalContent>
         <ModalHeader>
-          <Flex flexDir="column" mt="40px" mb="20px">
-            <Text fontSize="xl" fontWeight="bold">
-              Log in or sign up
+          <Flex flexDir="column" alignItems="center" mt="40px">
+            <Text sx={logoStyles} fontSize="2xl" mb="20px" role="logo">
```

Comment:
Can we use variant here？
<img width="832" alt="image" src="https://user-images.githubusercontent.com/61776449/173228836-2d8f2a60-01fe-43a4-8e3d-786cceac6244.png">
https://chakra-ui.com/docs/components/typography/text

Reply:
logoStyles was not used as logo text was replaced with svg.